instances/technique/activationLikelihoodEstimation.jsonld

```diff
@@ -0,0 +1,20 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/activationLikelihoodEstimation",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "An 'activation likelihood estimation' is a coordinate-based meta-analysis of neuroimaging data that determines the above-chance convergence of activation probabilities between experiments (i.e., not between foci). [adapted from [Eickhoff et al., 2011](https://dx.doi.org/10.1016%2Fj.neuroimage.2011.09.017)]",
```

Comment:
I think here is one bracket too much
`ci). [adapted from`

Reply:
no that is on purpose. (square brackets around the "adapted from  Eickhoff et al., 2011" where "Eickhoff et al., 2011" is then a link

Reply:
@skoehnen or did I miss something?

Reply:
Okay, just making sure :)